### DISCUSSION TITLE
Every time I delete a file, Phoenix says there was an external change and prompts me

### DISCUSSION THREAD

**Discussion by treeguy1234** [phcode-dev/phoenix#1892] [Q&A] [answered]
Example: 

https://github.com/user-attachments/assets/8ee6b979-9d20-48ff-914d-9f938e3404bb

**Comment by abose** [comment]
Which os is this? 
Is this the browser app or desktop app?
Is there anything special about the folder like is it a network drive?

**Reply by treeguy1234** [reply]
My computer's OS is Windows 11 and I use the browser app. It is connected to Google Drive via Google Drive for desktop. Thank you for taking the time to help resolve my issue👍

**Reply by abose** [reply, marked as answer]
Got it, this is a known issue with google drive connected folders.

Please track in https://github.com/phcode-dev/phoenix/issues/1710

We do not have a planned fix date for this yet, apologies for the issue.
